workflow/rocoto/gfs_tasks.py

```diff
@@ -950,10 +950,10 @@ def _fcst_cycled(self):
             dependencies.append(rocoto.add_dependency(dep_dict))
             dependencies = rocoto.create_dependency(dep_condition='or', dep=dependencies)
 
-        if self.options['do_wave']:
-            dep_dict = {'type': 'task', 'name': f'{self.run}_waveprep'}
-            dependencies.append(rocoto.add_dependency(dep_dict))
-            dependencies = rocoto.create_dependency(dep_condition='and', dep=dependencies)
+        #if self.options['do_wave']:
+        #    dep_dict = {'type': 'task', 'name': f'{self.run}_waveprep'}
+        #    dependencies.append(rocoto.add_dependency(dep_dict))
+        #    dependencies = rocoto.create_dependency(dep_condition='and', dep=dependencies)
```

Comment:
Suggest removing unless this will be added back later, in which case a `TODO` should be added.
```suggestion
```